store/src/main/java/com/dropbox/android/external/store4/ExperimentalStoreApi.kt

```diff
@@ -0,0 +1,10 @@
+package com.dropbox.android.external.store4
```

Comment:
i dont think this helps much as no IDE will understand it.
should we instead use the Experimental from AndroidX?

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:annotation/annotation-experimental/src/main/java/androidx/annotation/experimental/Experimental.kt;l=35?q=experimental&ss=androidx

Reply:
hmm I thought the AndroidX one is for Java support?

Here's the IDE warning I get from the call side without `@ExperimentalStoreApi`:

<img width="712" alt="experiment-store-ide-warning" src="https://user-images.githubusercontent.com/11519072/73130227-8ae04d00-4048-11ea-89b3-236b56d526fd.png">

What am I missing?

Reply:
oh nevermind, looks like this is some standard thing, i'm wrong. sorry.